loadbalancer/fadein_test.go

```diff
@@ -326,14 +327,15 @@ func benchmarkFadeIn(
 			LBFadeInDuration: fadeInDuration,
 			LBFadeInExponent: 1,
 		}
-		registry := routing.NewEndpointRegistry(routing.RegistryOptions{})
+		detected := map[string]time.Time{}
 		for i := range eps {
 			route.LBEndpoints = append(route.LBEndpoints, routing.LBEndpoint{
 				Host:     eps[i],
 				Detected: detectionTimes[i],
 			})
-			registry.SetDetectedTime(eps[i], detectionTimes[i])
+			detected[eps[i]] = detectionTimes[i]
 		}
+		registry := routing.NewEndpointRegistry(routing.RegistryOptions{Detected: detected})
```

Comment:
Do we need this complication?
Maybe we should just export SetDetected and SetLastSeen?

Reply:
Exported them.